src/views/Dashboard.vue

```diff
@@ -43,19 +43,17 @@
             <section id="dashboard-filing-history-section">
               <header>
                 <h2>
-                  <span>Recent Filing History</span>&nbsp;<span class="gray6">({{historyCount}})</span>
+                  <span>Recent Filing History</span>&nbsp;<span class="gray6">({{getHistoryCount}})</span>
                 </h2>
                 <StaffNotation
                   v-if="isRoleStaff && !!businessId"
                   addScrollbarOffset="true"
-                  :disabled="!isAllowed(AllowableActions.FILE_STAFF_NOTATION)"
                   @close="reloadDataIfNeeded($event)"
                 />
               </header>
               <FilingHistoryList
                 class="mt-3"
                 :highlightId="filingId"
-                @history-count="historyCount = $event"
               />
```

Comment:
What do you think of passing the `businessId` as prop into `FilingHistoryList`?  Since `businessId` is required, we should be explicit and show what's needed.

Reply:
This also fits with our long-term plan of having `businessId` become a route parameter that's passed from VueRouter to the view as a prop.

Reply:
Yes, I like being explicit about what parameters a component needs (eg, props).

Again, businessId doesn't change once the app has started, so why parameterize it? But it would be nice if it came from somewhere central and internal (like the store, not session storage).

Open to other ideas here, but I think that should be handled in a different (new) ticket.

Reply:
> This also fits with our long-term plan of having `businessId` become a route parameter that's passed from VueRouter to the view as a prop.

Sure, but businessId never changes, so why parameterize it? (Other than for visibility.) I mean, the FHL will never run on its own or in a new app, so why can't we consider the businessId to be pretty much static (like, say, the environment variables).